FABulous/FABulous_CLI/FABulous_CLI.py

```
@@ -271,6 +277,7 @@ def do_load_fabric(self, args):
                 logger.error(
                     "No argument is given and the csv file is set or the file does not exist"
                 )
+                return
```

Should we throw an error here?

---

Ok, then we should keep it that way. :)